R-package/configure.ac

```diff
@@ -4,6 +4,9 @@ AC_PREREQ(2.62)
 
 AC_INIT([xgboost],[0.6-3],[],[xgboost],[])
 
+# Use this line to set CC variable to a C compiler
+AC_PROG_CC
```

Comment:
?  Why ?

Reply:
The feature testing script needs access to `CC` variable.

Reply:
@hcho3 Yup, but why setting it to a `C` complier?  I assume that means changing `g++` to `gcc` or `clang++` to `clang`.

Reply:
`CC` refers to a C compiler and `CXX` refers to a C++ compiler.

Reply:
Yup.   I thought we use `CXX` var exclusively so are compilation tests ... Never mind,  I don't know anything about autotools.  Sorry for the noise.

Reply:
Don't worry, I don't know much about autotools either. I wait for the day where we can use CMake for CRAN submissions.